MoEngagePluginCards/MoEngagePluginCardsBridge.swift

```diff
@@ -0,0 +1,422 @@
+//
+//  MoEngagePluginCardsBridge.swift
+//  MoEngagePluginCards
+//
+//  Created by Soumya Mahunt on 19/06/23.
+//
+
+import MoEngagePluginBase
+import MoEngageCore
+import MoEngageCards
+
+@objc public protocol MoEngageCardSyncDelegate {
+    func syncComplete(forEventType eventType: MoEngageCardsSyncEventType, withData data: [String: Any])
```

Comment:
@msoumya-engg-sdk it should be json as per contracts

Reply:
@RakshithaAcharya the `data` parameter contains JSON as per contract. The `eventType` `MoEngageCardsSyncEventType` is sent for native wrapper to determine equivalent flutter method to invoke in channel.